frontend/javascripts/oxalis/model/accessors/dataset_layer_transformation_accessor.ts

```diff
@@ -402,7 +402,11 @@ function hasValidTransformationPattern(transformations: CoordinateTransformation
 }
 
 function _doAllLayersHaveTheSameRotation(dataLayers: Array<APIDataLayer>): boolean {
-  const firstDataLayerTransformations = dataLayers[0]?.coordinateTransformations;
+  if (dataLayers.length === 0) {
+    // The dataset does not have any layers. Therefore no layers can be rotated.
+    return false;
+  }
+  const firstDataLayerTransformations = dataLayers[0].coordinateTransformations;
```

Comment:
And here I noticed while thinking about this bug, that the previous code would return true for `_doAllLayersHaveTheSameRotation` when it was called with an empty array of data layers. This seemed wrong to me on a conceptual level as one cannot rotate something that has no data. Feel free to disagree :)